src/packer.pkr.hcl

```
@@ -110,7 +110,7 @@ source "amazon-ebs" "example" {
 }
 
 build {
-  sources = ["source.amazon-ebs.example"]
+  sources = ["source.amazon-ebs.egress-assess"]
```

To maintain consistency with my previous suggestion in this file:
```suggestion
  sources = ["source.amazon-ebs.egress_assess"]
```